client/my-sites/domains/domain-management/list/item.jsx

```diff
@@ -83,6 +121,9 @@ class ListItem extends React.PureComponent {
 	}
 
 	handleClick = () => {
+		if ( this.props.shouldUpgradeToMakePrimary ) {
```

Comment:
I think this is where we're eating/ignoring the click event when `this.props.enableSelection === true` -- as written, we *never* handle the click event, regardless of whether we're in select mode or not.

Reply:
Eh... there was a merge conflict changing how this stuff works, so this was an untested quick change before pushing.